int64 xPos
int64 value
int64 length
int64 hight
float32 velocity
float32 z
time timestamp
